docs/Air.md

```diff
@@ -434,3 +435,19 @@ Gets pnr information and tickets list from [`importPNR`](#importPNR) and then do
 | ignoreTickets | `Boolean` | Defines if tickets should be ignored. The default value is `false` |
 
 **See: <a href="../examples/Air/cancelBooking.js">cancelBooking example</a>**
+
+## .addSegments(params)
+<a name="addSegments"></a>
+
+Add segments to an existing reservation record.
+
+**Returns**: `Promise` which is resolved with response message.
+
+| Param | Type | Description |
+| --- | --- | --- |
+| pnr | `String` | PNR |
+| version | `Number` | Current uAPI record version |
+| universalRecordLocatorCode | `String` | uAPI universal locator code |
+| reservationLocatorCode | `String` | uAPI reservation locator code |
+| bookingTravelerRef | `String` | uAPI booking traveler reference key |
```

Comment:
Have you tried adding segments to the PNR with more than one passenger?
I believe, segments could not be added per passenger.
Please check and take not the following actions:
- If multiple references should be provided, change type to array
- If one reference could be provided, please report

Either way if we could retrieve it by PNR as well as data in previous comment, would be fine to keep this params optional

Reply:
Figured out it's an optional parameter and can be safely skipped, I'll create a new booking and check it to confirm.